jupyterlab_caip_optimizer/src/store/view.ts

```diff
@@ -19,9 +19,13 @@ export const viewSlice = createSlice({
   } as ViewState,
   reducers: {
     setView: (state, action: PayloadAction<ViewType>) => {
-      if (action.payload.view === 'studyDetails') {
+      if (
```

Comment:
I cleaned this up on my branch. It should make this reducer less messy.

Reply:
Then I guess I'll leave this as it is for now, and can be cleaned in your PR?